share/spice/rand_word/rand_word.js

```diff
@@ -34,23 +31,34 @@
                 }
             }
         };
-        
-        if ( list_of_list[0].length == 1 ) {
-            spiceObj.data = {
-                title: "Random Word",
-                word: first.word
-            };
-            
-            spiceObj.templates = {
-                group: 'text',
-                options: {
-                    content: Spice.rand_word.single_word,
-                    moreAt: true
-                }
-            };
+        if (list_of_list[0].length == 1) {
+            $.getJSON("/js/spice/rand_word_fetch_id/" + first.word, function(data) {
+                var first = data[0];
+                $.extend(spiceObj, {
+                    data: {
+                        title: "Random Word",
```

Comment:
@VeerpalBrar let's make the title the actual random word

We can use a subtitle to specify it was a randomly chosen word